app/views/layouts/govuk_publishing_components/application.html.erb

```diff
@@ -1,19 +1,24 @@
 <% content_for :body do %>
   <% if @preview %>
-    <main id="wrapper" class="govuk-width-container">
-      <%= yield %>
-    </main>
+    <% 
+      govuk_container_class = @component_doc.capture_full_width_screenshot? ? "govuk-full-width-container" : "govuk-width-container"
+    %>
+    <%= tag.div id: "wrapper", class: govuk_container_class do %>
```

Comment:
For components that use the new full width, would it be worth putting a little bit of spacing between the edge of the screen and the preview name?

<img width="585" height="159" alt="Screenshot 2025-08-04 at 08 51 09" src="https://github.com/user-attachments/assets/27990e0e-4706-4ee9-a533-a278f2c55291" />

Reply:
Thanks for the review, I've fixed the typo on the 2nd commit description and also increased the spacing between the component title and edge of the screen when the preview page uses the full width container.